pom.xml

```diff
@@ -169,6 +198,28 @@
                     <plugin>
                         <groupId>org.codehaus.mojo</groupId>
                         <artifactId>rpm-maven-plugin</artifactId>
+                        <executions>
+                            <execution>
+                                <id>attach-rpm</id>
+                                <goals>
+                                    <goal>attached-rpm</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <group>Applications/Archiving</group>
```

Comment:
The lic-folder should be in `/etc/${dans-provider-name}/${project.artifactId}/lic`. Also make sure to add `<configuration>${rpm-replace-configuration}</configuration>`, in order to mark them as configuration files to RPM.

Reply:
Done. Except that the folder is /etc/**opt/**${dans-provider-name}/${project.artifactId}/lic

There was a lot of confusion first in trying to implement this change.  It appeared that when changing the directory to `/etc/${dans-provider-name}/${project.artifactId}/lic`, licenses didn't appear into the `rpm`.  They did appear into rpm when something else was given as the target directory, e.g. /etc/**xxx**/${dans-provider-name}/${project.artifactId}/lic.

I was wondering if this had to do with the fact that in the parent pom there was already a mapping to `/etc/opt/${dans-provider-name}/${project.artifactId}`. I tried all kinds of combinations of `<mapping>` attributes in `easy-validate-dans-bag` , but to no avail.

In the end I decided to modify the parent pom. After trying something out, I decided to try one more thing with the original parent pom from the repository.  And voila, it worked!  

Now it turned out that making changes in the mapping of easy-validate-dans-bag didn't trigger a change in rpm in easy-validate-dans-bag `target` directory.  Only after calling another parent pom and then calling again the original parent pom, the changes were seen in the rpm.